cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +202,81 @@
+# For the full ACME spec, see:
+# 	https://tools.ietf.org/html/draft-ietf-acme-acme-02
+# 	https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html
+[SXG.ACME]
+  # This is the ACME discovery URL that is used for ACME http requests to the
+  # Certificate Authority that doles out the certificates. Currently, the only
+  # CA that supports automatic signed exchange cert renewals is DigiCert:
+  # https://docs.digicert.com/certificate-tools/acme-user-guide/acme-directory-urls-signed-http-exchange-certificates/
+  DiscoveryURL = "https://production-acme.discovery.url/"
+
+  # This is the email address you used to create an account with the Certificate
+  # Authority that is registered to request signed exchange certificates.
+  Email = "user@company.com"
+
+  # For the remaining configuration items, it's important to understand the
+  # different challenges employed as part of the ACME protocol:
+  # https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html#identifier-validation-challenges
+  # https://letsencrypt.org/docs/challenge-types/
+  # https://certbot.eff.org/docs/challenges.html?highlight=http
+  #
+  # Note that you don't need to have all the challenges configured, it's
+  # typically sufficient to have one configured. The exception arises when you
+  # have to deal with wildcard certificates, see below.
+
+  # This is the http server root directory where the ACME http challenge token
+  # could be deposited. Note that you may need to do some configuration work to
+  # get this setup to work where multiple instances of AMP Packager is running.
+  # For example:
+  # https://community.letsencrypt.org/t/how-to-nginx-configuration-to-enable-acme-challenge-support-on-all-http-virtual-hosts/5622/3
+  HttpWebRootDir = '/path/to/www_root_dir'
+
+  # This is the port used by the AMP Packager to respond to the HTTP challenge
+  # issued as part of ACME protocol. Note that if your setup only opens up
+  # certain ports, you may need to do a configuration change where you forward
+  # requests to this port using proxy_pass, for example:
+  # https://medium.com/@dipeshwagle/add-https-using-lets-encrypt-to-nginx-configured-as-a-reverse-proxy-on-ubuntu-b4455a729176
+  HttpChallengePort = 5002
+
+  # This is the port used by AMP packager to respond to the TLS challenge issued
+  # as part of the ACME protocol.
+  TlsChallengePort = 5003
+
+  # This is the DnsProvider to be used in fulfilling the ACME DNS challenge.
+  # Note that you only need the DNS challenge setup if you have wildcard
+  # certificates: https://searchsecurity.techtarget.com/definition/wildcard-certificate
+  # For the DNS challenge, go-acme/lego, there are certain environment variables
+  # that need to be set up which depends on the DNS provider that you use to
+  # fulfill the DNS challenge: https://go-acme.github.io/lego/dns/
+  # To use this, build amppkg with `go build -tags dns01`; it is disabled by
+  # default because it bloats the binary.
+  #DnsProvider = "gcloud"
```

Comment:
This config file states at the beginning: "Commented out lines show the default value for each parameter." It doesn't seem you've set the default value in the code though.

Reply:
Fixed, DNSProvider should default to empty string.